scrapy/utils/gz.py

```diff
@@ -20,9 +14,11 @@
 # - scrapy does not support Python 3.2
 # - Python 2.7 GzipFile works fine with standard read() + extrabuf
 if six.PY2:
+    from cStringIO import StringIO as BytesIO
```

Comment:
I think now it reads like BytesIO is imported because of GzipFile.read problem described above, while it doesn't.

Reply:
I didn't realise that, thank you for bringing it to my notice.

Reply:
hey @sbs2001! I don't think it is solved: read1 functions work aroung gzip problems, and comment should relate to them, not to io imports